EmojicodeCompiler/Lexer.cpp

```diff
@@ -133,7 +134,7 @@ TokenStream lex(const char *path) {
                     }
                     continue;
                 case TokenType::Variable:
-                    if (detectWhitespace(c, &sourcePosition.character, &sourcePosition.line)) {
+                    if ((c != 0xFE0F) && detectWhitespace(c, &sourcePosition.character, &sourcePosition.line)) {
```

Comment:
That sounds rather suspicious... and not intended. What happened to the s package? I think I’ll have to have a look at this.